src/main/java/cz/it4i/parallel/Constants.java

```diff
@@ -0,0 +1,11 @@
+package cz.it4i.parallel;
+
+interface Constants {
+	long TEMPLATE_ID = 6;
+	int TIMEOUT = 3600;
+	long CLUSTER_NODE_TYPE = 7l;
+	String PROJECT_ID = "open-12-20";
+	String CONFIG_FILE_NAME = "configuration.properties";
+	String JOB_NAME = "ImageJ-Server-Heappee";
```

Comment:
It'd be good to distinguish what's applicable for the entire package (e.g. configuration.properties?) and what's specific for a given paradigm (e.g. job name "ImageJ-Server-Heappee" [sic!, there is a typo btw]).

Consequently, consider moving heappe-paradigm-specific constants to the paradigm class.

Reply:
OIK